### PR TITLE
Allow Origin header with capital O

### PR DESCRIPTION
**Pull request by sschneider1207**
This fixes (some?) instances of #18

### CODE REVIEW

**Comment by sschneider1207** [comment]
@mschae do you have any thoughts on this?

File: lib/cors_plug.ex [hunk 1]
```diff
@@ -73,7 +73,8 @@ defmodule CORSPlug do
   # return request origin if in origin list, otherwise "null" string
   # see: https://www.w3.org/TR/cors/#access-control-allow-origin-response-header
   defp origin(origins, conn) when is_list(origins) do
-    req_origin = get_req_header(conn, "origin") |> List.first
+    combined_origin_headers = get_req_header(conn, "origin") ++ get_req_header(conn, "Origin")
+    req_origin = List.first(combined_origin_headers)
```
Comment (mschae) [inline lib/cors_plug.ex]: Sorry I'd like the equal signs to be aligned.

How about this?

```Elixir
alias Plug.Conn
# ...
defp origin(origins, %Conn{req_headers: headers}) when is_list(origins) do
  req_origin = List.find(headers, fn({k, v}) String.downcase(k) == origin end)
  if req_origin in origins, do: req_origin, else: "null"
end

**Comment by mschae** [comment]
Sorry, this completely escaped me! I'd like to make this a little easier readable, please review the changes.

Thanks for contributing!

**Comment by sschneider1207** [comment]
@mschae I updated this to be more similar to your suggestion.

**Comment by mschae** [comment]
Thanks, this is merged, I will release asap.

**Comment by somlor** [comment]
FWIW, as of around Plug 1.1 all headers are automatically downcased https://hexdocs.pm/plug/Plug.Conn.html#module-request-fields

**Pull request merged**